README.md

```diff
@@ -1,104 +1,106 @@
+* [DigitalOcean](https://www.digitalocean.com/), cloud provider without headaches
+* [Ansible](https://github.com/ansible/ansible), keeps all your servers in sync without manual work
+* [Terraform](https://github.com/hashicorp/terraform), create your digital infrastructure by declaring it in code
+* [Docker](https://github.com/docker/docker-ce), containerization platform
+* [Kubernetes (K8S)](https://github.com/kubernetes/kubernetes), takes away lot's of hosting headaches
 
 ### Databases and caching
 
-* PostgreSQL, our relational-database of choice
-* MySQL, populair alternative to PostgreSQL
-* Redis, key-value in-memory database, great for caching
-* Varnish, HTTP accelerator for content-heavy dynamic web sites and APIs
+* [PostgreSQL](https://github.com/postgres/postgres), our relational-database of choice
+* [Redis](https://github.com/antirez/redis), key-value in-memory database, great for caching
+* [Varnish](https://github.com/varnishcache/varnish-cache), HTTP accelerator for content-heavy dynamic web sites and APIs
 
 ### Methodologies
 
-* SemVer, keeping track of your software versioning
-* 12-factor app, the process of development how it should be
-* SOLID and DRY, keeping your code maintainable
-* TDD, test-driven development to ensure working software
-* DevSecOps, combining development, security and operations
-* Site-reliability engineering, taking devops a setp further
-* Async Manifesto, a 21st century successor to Agile
+* [SemVer](https://semver.org/), keeping track of your software versioning
+* [12-factor app](https://12factor.net/), the process of development how it should be
+* [SOLID](https://en.wikipedia.org/wiki/SOLID) and [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), keeping your code maintainable
+* [TDD](https://en.wikipedia.org/wiki/Test-driven_development), test-driven development to ensure working software
+* [DevSecOps](https://www.redhat.com/en/topics/devops/what-is-devsecops), combining development, security and operations
+* [Site-reliability engineering (SRE)](https://en.wikipedia.org/wiki/Site_Reliability_Engineering), taking devops a setp further
+* [Async Manifesto](http://asyncmanifesto.org/), a 21st century successor to Agile
 
-## Design resources
+## Design/image resources
 
-* Undraw, open-source illustrations
-* Humaaans, open-source illustrations
-* Dribble, platform where artists show off their work
-* Behance, platform where artists show off their work
+* [Undraw](https://undraw.co/), open-source illustrations
+* [Humaaans](https://www.humaaans.com/), open-source illustrations
+* [Dribble](https://dribbble.com/), platform where artists show off their work
+* [Behance](https://www.behance.net/), platform where artists show off their work
+* [Pexels](https://www.pexels.com/) and [StockSnap](https://stocksnap.io/), free to use stock images
+* [PlaceKitten](http://placekitten.com/), kitten placeholder images
 
```

Comment:
Maybe mention invision?

Reply:
I considered it, but it's a different tier of software. Might add it to a separate category later on though.